src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -313,47 +311,54 @@ const HealthCardDisplay = () => {
 
   return (
     <Grid container className={styles.root}>
-      {(qrError)
-        ? (
-          <ErrorFallback error={qrError} />
-        ) : (
-          <ErrorBoundary FallbackComponent={ErrorFallback}>
-            <HealthCardDataProvider healthCardJws={jws[jws.length - 1]}>
-              <Banners />
-            </HealthCardDataProvider>
-            <Grid item className={styles.flexCard}>
-              <Box style={{ textAlign: 'center' }}>
-                <Router>
-                  <Link
-                    to="/qr-scan"
-                    state="link"
-                    replace
-                    onClick={() => handleScan('link')}
-                  >
-                    {t(
-                      'healthcarddisplay.Add another SMART Health Card for same person'
-                    )}
-                  </Link>
-                </Router>
-              </Box>
-              {bannersUpdated && (
+      {qrError ? (
+        <ErrorFallback error={qrError} />
+      ) : (
+        <ErrorBoundary FallbackComponent={ErrorFallback}>
+          <HealthCardDataProvider healthCardJws={jws[jws.length - 1]}>
+            <Banners />
+          </HealthCardDataProvider>
+          <Grid item className={styles.flexCard}>
+            <Box className={styles.mismatchDemographicDataText}>
+              {matchingDemographicData ? (
+                ''
+              ) : (
                 <>
-                  {
-                    // Display cards in reverse order that they were scanned
-                    jws.slice(0).reverse().map((hcJws) => (
+                  <Box>
```

Comment:
I don't love this on top centered. Might look better to the left of the text. We can run it by the group.